src/components/index.ts

```diff
@@ -33,6 +33,8 @@ export { default as Tooltip } from './tooltip';
 export { default as Accordion } from './accordion';
 export { default as AccordionButton } from './accordion/AccordionButton';
 export { default as AccordionPanel } from './accordion/AccordionPanel';
+export { default as AccordionMenu } from './accordion/AccordionMenu';
```

Comment:
shouldn't this component only be used inside components-pkg for the header/footer? I wouldn't export it if that is the case